src/hotel/booking/BookingCTL.java

```diff
@@ -16,7 +16,7 @@
 public class BookingCTL {
 	
 	
-	private static enum State {PHONE, ROOM, REGISTER, TIMES, CREDIT, APPROVED, CANCELLED, COMPLETED}	
+	public static enum State {PHONE, ROOM, REGISTER, TIMES, CREDIT, APPROVED, CANCELLED, COMPLETED}
```

Comment:
enum State can be set back to private - post testing.

Reply:
resolved